main.tf

```
@@ -7,12 +7,28 @@ locals {
   name = "new_account_support_case_${random_string.id.result}"
 }
 
+
+data "aws_partition" "current" {}
+
+data "aws_iam_policy_document" "lambda" {
+  statement {
+    actions = [
+      "organizations:DescribeCreateAccountStatus"
```

Inspecting the lambda code, this will also need some `support` permissions:

```suggestion
      "organizations:DescribeCreateAccountStatus",
      "support:CreateCase",
      "support:DescribeCases",
```

---

Thanks for catching that.

---

They are used by the lambda itself:

* https://github.com/plus3it/terraform-aws-org-new-account-support-case/blob/939d5c7aa58094967dd96905545baca47ad12a92/lambda/src/new_account_support_case.py#L104
* https://github.com/plus3it/terraform-aws-org-new-account-support-case/blob/master/lambda/src/new_account_support_case.py#L125

---

My apologies - you're correct.  I confused them with the `organizations` calls I'm making in the unit tests, not the `support` service.  Corrections made.